src/System.Collections/tests/System.Collections.Tests.csproj

```diff
@@ -2,6 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
+    <!-- Codegen bug, VSTS tracking number 472947 -->
+    <ILCBuildType Condition="'$(TargetGroup)' == 'uap'">chk</ILCBuildType>
```

Comment:
This condition will never be true as this test project doesn't have a uap or uapaot configuration and it will cause a regression on the tests. Could you please remove the condition? 

https://github.com/dotnet/corefx/blob/master/src/System.Collections/tests/Configurations.props#L5-L6